packages/teams-js/src/public/chat.ts

```diff
@@ -79,7 +81,7 @@ export namespace chat {
         );
       } else {
         const sendPromise = sendAndHandleStatusAndReason('chat.openChat', {
-          members: openChatRequest.user,
+          members: [openChatRequest.user],
```

Comment:
>members: [openChatRequest.user], [](http://example.com/codeflow?start=10&length=32)

This is the change that ultimately fixes the bug: the `members` property of the `chat.openChat` message is expected to always be an array, even if there's only one user.